lib/Serverless.test.js

```diff
@@ -207,9 +206,10 @@ describe('Serverless', () => {
         .stub(serverless.variables, 'populateService').resolves();
       runStub = sinon
         .stub(serverless.pluginManager, 'run').resolves();
-    });
+    }));
 
     afterEach(() => {
+      if (!serverless.utils.logStat.restore) return;
```

Comment:
Why is this needed?

Reply:
As `serverless.init()` fails, stubs are not initialized therefore there's no `restore` methods.
Without this, there will be additional crash in `afterEach`.

Still it's indeed not needed when `serverless.init()` succeeds, so assuming we're fix it, we can indeed remove it, as it's just extra noise